CONTRIBUTING.md

```diff
@@ -0,0 +1,258 @@
+- [Introduction](#introduction)
+- [The contribution process](#the-contribution-process)
+  - [Raise an issue](#raise-an-issue)
+  - [Create a fork](#create-a-fork)
+  - [Create a new folder](#create-a-new-folder)
+  - [Add license](#add-license)
+  - [Create a notebook](#create-a-notebook)
+  - [Commit new changes](#commit-new-changes)
+  - [Open a pull request](#open-a-pull-request)
+  - [Common recommendations for the review](#common-recommendations-for-the-review)
+- [CI/CD test passing guide](#cicd-test-passing-guide)
+  - [PEP 8 Style](#pep-8-style)
+  - [Notebook execution](#notebook-execution)
+  - [Format requirements](#format-requirements)
+- [Benchmarking result report](#benchmarking-result-report)
+
+## Introduction
+
+Thanks for considering a contribution to the MONAI Tutorials Repository and reading the Contributing Guidelines.
+
+The following is a set of guidelines for contributing tutorials to Project MONAI.
+We want to use the guideline to support the growth of the project, instead of restricting the depth and breadth of your contribution.
```

Comment:
```suggestion
We want the guidelines to support the growth of the project, instead of restricting the depth and breadth of your contribution.
```

Reply:
Fixed as suggested. Thank you @ericspod !